pootle/templates/staticpages/_body.html

```diff
@@ -9,7 +9,14 @@
   {% endblocktrans %}
   </div>
   {% endif %}
-  <h1 class="title">{{ page.title }}</h1>
+  <h1 class="title">
+    {{ page.title }}
+    {% if user.is_superuser %}
```

Comment:
After discussion on #3908, and given that the announcements are only displayed on TPs, it is ok to display the edit link if the user is a superuser or has `administrate` permission in the current TP.